### ISSUE TITLE
New Data source - Macedon Ranges, Victoria, Australia

### ISSUE THREAD

**Issue by GooeyLizard9000** [mampfes/hacs_waste_collection_schedule#159]
Hoping there's an easy way to add my local council waste days as a data source.
There's an address lookup page here.
https://www.mrsc.vic.gov.au/Live-Work/Bins-Rubbish-Recycling/Bins-and-collection-days/Bin-collection-days

I've just got no idea about how to even start looking at being able to get the data to connect.

Thanks
Simon

**Comment by tathamoddie** [comment]
The [Macedon search page](https://www.mrsc.vic.gov.au/Live-Work/Bins-Rubbish-Recycling/Bins-and-collection-days/Bin-collection-days) looks nearly identical to [the Stonnington page](https://www.stonnington.vic.gov.au/Services/Waste-and-recycling)

I implemented a Stonnington provider here: https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/custom_components/waste_collection_schedule/waste_collection_schedule/source/stonnington_vic_gov_au.py

You could model off that

**Comment by tathamoddie** [comment]
I've implemented a pull request for this (#162). Once @mampfes is able to review it, you should see this issue get closed once, and see Macedon Ranges support working.

If you want to try it out earlier, you can hack the change in locally:

1. Download https://raw.githubusercontent.com/mampfes/hacs_waste_collection_schedule/87476a2ffd62a8deb654962ed3bc1fd4632ac51d/custom_components/waste_collection_schedule/waste_collection_schedule/source/mrsc_vic_gov_au.py
1. Drop that into your local HA instance, in `config\custom_components\waste_collection_schedule\waste_collection_schedule\source`
1. Put this in your `configuration.yaml`:

    ```
    waste_collection_schedule:
      sources:
        - name: mrsc_vic_gov_au
          args:
            street_address: 45 Aitken Street, Gisborne

    sensor waste:
      - platform: waste_collection_schedule
        name: next_collection
        add_days_to: true
        details_format: generic
        value_template: '{{value.types|join(", ")}}'
    ```
1. Restart Home Assistant
1. Look for a `sensor.next_collection` entity showing up after a minute or two

**Comment by GooeyLizard9000** [comment]
Thank you, added the mrsc source and it works perfectly.
